specifications/api/swagger.yaml

```diff
@@ -433,6 +433,133 @@ x-settings: &settings
 
         After update existing metadata will be dropped forcing it to re-sync automatically with the new setting.
 
+x-assetPolicyItem: &assetPolicyItem
+  type: string
+  description: |
+    The asset on-chain type which acts as a sub-identifier within a policy.
+
+    For policies with a single fungible asset item, this is typically
+    an empty string.
+
+    This value can be up to 32 bytes of arbitrary data (which is 64
+    hexadecimal digits).
+  # fixme: (remove this note) I changed it to hex from base64, the reason is below.
+  format: hex
+  maxLength: 64
+  # fixme: (remove this note) multiple examples seem not to validate
```

Comment:
Unfortunately no, OpenAPI does not support multiple examples... I think there's a Redoc extension we could use if we really wanted that, but it's a vendor-specific extension so not a pure OpenAPI primitive.